apps/testing/lib/decoder/noop.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Decoder.Noop do
```

Comment:
```suggestion
defmodule Decoder.Fake do
```
Why? Why didn't you name it consistently?!

Reply:
lets go with, I did it on purpose to drive you nuts.  But in reality I just didn't think of it.